code/modules/mob/living/carbon/human/species_types/oozelings.dm

```diff
@@ -141,20 +142,20 @@
 		to_chat(H, "<span class='notice'>You feel intact enough as it is.</span>")
 		return
 	to_chat(H, "<span class='notice'>You focus intently on your missing [limbs_to_heal.len >= 2 ? "limbs" : "limb"]...</span>")
-	if(H.blood_volume >= 80*limbs_to_heal.len+BLOOD_VOLUME_OKAY)
+	if(H.blood_volume >= (BLOOD_VOLUME_SAFE - BLOOD_VOLUME_OKAY)*limbs_to_heal.len+BLOOD_VOLUME_OKAY)
 		if(do_after(H, 60, target = H))
 			H.regenerate_limbs()
-			H.blood_volume -= 80*limbs_to_heal.len
+			H.blood_volume -= (BLOOD_VOLUME_SAFE - BLOOD_VOLUME_OKAY)*limbs_to_heal.len
 			H.nutrition -= 20*limbs_to_heal.len
 			to_chat(H, "<span class='notice'>...and after a moment you finish reforming!</span>")
 		return
-	if(H.blood_volume >= 80)//We can partially heal some limbs
-		while(H.blood_volume >= BLOOD_VOLUME_OKAY+80 && LAZYLEN(limbs_to_heal))
+	if(H.blood_volume >= (BLOOD_VOLUME_SAFE - BLOOD_VOLUME_OKAY))//We can partially heal some limbs
+		while(H.blood_volume >= BLOOD_VOLUME_OKAY+(BLOOD_VOLUME_SAFE - BLOOD_VOLUME_OKAY) && LAZYLEN(limbs_to_heal))
```

Comment:
```suggestion
		while(H.blood_volume >= BLOOD_VOLUME_SAFE && LAZYLEN(limbs_to_heal))
```
this does nothing mathematically: 1 + 2 - 1 = 2

Reply:
This is logically incorrect and needs to be reverted.